substra/cli/interface.py

```diff
@@ -112,6 +112,15 @@ def click_global_conf_config(f):
         help='Config path (default ~/.substra).')(f)
 
 
+def click_global_conf_metadata(f):
+    """Add metadata option to command."""
+    return click.option(
+        '--metadata',
+        callback=update_global_conf,
+        type=click.Path(exists=True, resolve_path=True),
+        help='Metadata file path')(f)
```

Comment:
Since metadata is a path, you may want to name and use it the same way we did for the other paths (`--data-samples-path` and `--out-trunk-model-permissions-path`): name it `--metadata-path`  and use the `load_json_from_path` callback.